R/combined_seasonal_output.R

```diff
@@ -0,0 +1,156 @@
+#' Compute seasonal onset and burden levels from seasonal time series observations.
+#'
+#' @description
+#'
+#' This function performs automated and early detection of seasonal epidemic onsets and calculates the burden
+#' levels of time series dataset stratified by season. The seasonal onset estimates growth rates for consecutive
+#' time intervals and calculates the sum of cases (sum_of_cases). The burden levels use the previous seasons to
```

Comment:
I suggest either wrapping in backticks
```suggestion
#' time intervals and calculates the sum of cases (`sum_of_cases`). The burden levels use the previous seasons to
```

or just removing, since the naming is so clear
```suggestion
#' time intervals and calculates the sum of cases. The burden levels use the previous seasons to
```